client/components/LearnerViewDirective.js

```diff
@@ -192,26 +205,52 @@ tie.directive('learnerView', [function() {
       </style>
     `,
     controller: [
-      '$scope', 'SolutionHandlerService', 'QuestionDataService',
+      '$scope', '$timeout', 'SolutionHandlerService', 'QuestionDataService',
       'LANGUAGE_PYTHON',
       function(
-          $scope, SolutionHandlerService, QuestionDataService,
+          $scope, $timeout, SolutionHandlerService, QuestionDataService,
           LANGUAGE_PYTHON) {
         var language = LANGUAGE_PYTHON;
-        var question = QuestionDataService.getData();
-        var prompts = question.getPrompts();
-        var currentPromptIndex = 0;
+        var questionSetId = 'strings';
```

Comment:
I think my benchmark was something along the lines of: if it's going to be in the project tracker list, it doesn't need to be in the code. However, I do like the (implicit) point about having the comment in close proximity to the code. I'll make the change.

I think this is also pointing to: let's also have a discussion in the next meeting about how we, as a team, want to keep track of TODOs.